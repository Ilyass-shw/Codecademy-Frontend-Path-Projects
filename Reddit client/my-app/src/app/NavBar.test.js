import { render } from "@testing-library/react";
import React from "react";
// import { NavBar } from "./NavBar.js";

test("<NavBar/> matches snapshot", ()=>{
    const component = render (
		<div className="sideNav">
			<ul>
				<a href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s">
					<li>icon</li>
				</a>

				<a href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s">
                    <li>icon</li>
					<li>Home</li>
				</a>

				<a href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s">
                <li>icon</li>
					<li>Trending</li>
				</a>

				<a href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s">
                <li>icon</li>
					<li>Subscriptions</li>
				</a>

				<a href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s">
                    <li>icon</li>
					<li>Library</li>
				</a>
			</ul>
		</div>
	);
console.log(component.container)
    expect(component.container).toMatchSnapshot()
})