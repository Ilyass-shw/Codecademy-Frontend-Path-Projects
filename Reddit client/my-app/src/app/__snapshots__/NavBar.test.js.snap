// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar/> matches snapshot 1`] = `
<div>
  <div
    class="sideNav"
  >
    <ul>
      <a
        href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s"
      >
        <li>
          icon
        </li>
      </a>
      <a
        href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s"
      >
        <li>
          icon
        </li>
        <li>
          Home
        </li>
      </a>
      <a
        href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s"
      >
        <li>
          icon
        </li>
        <li>
          Trending
        </li>
      </a>
      <a
        href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s"
      >
        <li>
          icon
        </li>
        <li>
          Subscriptions
        </li>
      </a>
      <a
        href="www.youtube.com/watch?v=3oUu7N65s6I&t=1860s"
      >
        <li>
          icon
        </li>
        <li>
          Library
        </li>
      </a>
    </ul>
  </div>
</div>
`;
